src/components/black-friday/payment/PriceCard.tsx

```diff
@@ -43,14 +43,14 @@ export default function PriceCard({
   };
 
   const convertedPrice = useMemo(() => {
-    if (country !== '€') {
+    if (currency !== '€') {
```

Comment:
Save this in an enum or union type, or you will have troubles in the future if we introduce new currencies.

```
type Currency = {
 EUR,
 USD
 ...
}
```

Also, I don't recommend you to check against the symbol, use an ID for the currency or similar (like EUR or USD)